bpf/lib/l3.h

```diff
@@ -88,6 +88,13 @@ static inline int ipv6_local_delivery(struct __sk_buff *skb, int l3_off, int l4_
 	skb->cb[CB_SRC_LABEL] = seclabel;
 	skb->cb[CB_IFINDEX] = ep->ifindex;
 
+	/*
+	 * Special LXC case for updating egress forwarding metrics.
+	 * Note that the packet could still be dropped but it would show up
+	 * as an ingress drop counter in metrics.
+	 */
+	update_data_metrics(skb->len, DIRECTION_EGRESS, 0);
```

Comment:
Here is where the enum reason makes things more readable vs '0'

Reply:
Is there any value in recording this special case as its own type LXC_FORWARD for example?

Reply:
0 is for forwarded. Not the direction. I can add a separate #define just for forwarded reason, to make it more readable.

Regarding `LXC_FORWARD` , I don't think we need that level of granularity yet from prometheus perspective, and hence I am using the first step for exposing total drops/forwards in each direction.


In prometheus, I envision to display total drops/drop reason and total drops/direction.
Similarly for forwards. I can make a note of this and do it as a separate PR if we decide to add more granularity going forward.

Let me know.

Reply:
Sounds good, we can add more granularity when/if its needed. No need to do it now. Maybe just a #define for FORWARD and I'm happy. Again just nit-picking.

Reply:
done